src/components/common/Extensions/ExtensionDetails/ExtensionDetailsAside.tsx

```diff
@@ -125,7 +128,10 @@ const ExtensionDetailsAside = ({
   return (
     <aside>
       <div className={styles.buttonWrapper}>
-        <ExtensionActionButton extensionData={extensionData} />
+        <ExtensionActionButton extensionData={extensionData} />{' '}
```

Comment:
Is the empty space (`{' '}`) needed here?

Reply:
No it isn't. I removed it.